src/createStore.js

```diff
@@ -104,9 +106,13 @@ export default function createStore(reducer, preloadedState, enhancer) {
     }
 
     let isSubscribed = true
+    let id = listenerID++
 
     ensureCanMutateNextListeners()
-    nextListeners.push(listener)
+    nextListeners.push({
```

Comment:
This is extra allocation for every single listener. I don't think this is an acceptable tradeoff.

Reply:
We need unique token to mark listeners. Do you have any suggestions? Thanks for your time!

Although I can use `store.subscribe(listener.bind({}))` to avoid this issue, I guess resolve it in redux internal is more friendly to users.